belt_nlp/test_bert_embedder_pooling.py

```diff
@@ -0,0 +1,68 @@
+        ]
+    }
+
+    belt_embeds = embed_doc(applications, belt_model)
+    print_embeddings_length(belt_embeds)
```

Comment:
This test would work better as a notebook.
We use the following distinction:
- In tests we put automatic tests to be run and see if there are any errors. There are asserts but no prints.
- In notebooks we put  instructive code to show how it works step by step. There are prints but no asserts.
This file looks to me like the second case.